src/components/styleable-images/ImageNetwork.tsx

```diff
@@ -0,0 +1,10 @@
+import NetworkImage from '../../images/network.svg'
+
+interface ImageNetworkProps extends React.HTMLProps<HTMLImageElement> {
+  crossOrigin?: '' | 'anonymous' | 'use-credentials' | undefined
+  className?: string
+}
+
+const ImageNetwork = (props: ImageNetworkProps) => <img {...props} src={NetworkImage} alt="Network image" />
```

Comment:
What's the rational behind creating a dedicated component for this image vs. importing the SVG directly?

Reply:
I could not figure out how to style the imported SVG with styled components, unless I wrap it with a React component and pass the className prop. Have you faced this issue before?

Reply:
I've just tested importing like that:

```ts
import NetworkSVG from '../images/svgs/network.svg'
```

And it works. 
I had to look at your Gatsby config to understand that SVGs have to be stored in the `svgs` folder in order to be imported as components. It's a bit opaque as you have svgs both in the `images` root and in the `svgs` folder. It may be great to make it a bit more consistent :)

Reply:
Yes, doing that way works. But sometimes I want to load the SVG with an `img` tag, cause if the DOM tree of an SVG is very deep, it causes low performance when loading the website.

Your comment made me rethink my approach and now I got rid of the dedicated `ImageNetwork.tsx` file and simply render the image tag where I need it. Thanks!